### PR TITLE
[Doc] - MON-16256 Retrieve alerts from kadiska 

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Target version (i.e. version that this PR changes)

- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] 22.10.x (staging)
- [ ] 23.04.x (staging)
- [ ] Cloud (staging)
- [x] Monitoring Connectors (staging)
- [ ] 23.10.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-kadiska-restapi.md [hunk 1]
```diff
@@ -3,116 +3,232 @@ id: applications-monitoring-kadiska-restapi
...
-|:----------------|:-------------------------------|
-| Runners         | Découverte de runners Kadiska  |
-| Watchers        | Découverte de watchers Kadiska |
+</TabItem>
+<TabItem value="App-Monitoring-Kadiska-Runner-Restapi-custom" label="App-Monitoring-Kadiska-Runner-Restapi-custom">
+
+| Alias             | Modèle de service                                       | Description                                                           | Découverte |
+|:------------------|:--------------------------------------------------------|:----------------------------------------------------------------------|:----------:|
+| Alerts            | App-Monitoring-Kadiska-Restapi-Alerts-custom            | Vérifie le status des règles d'alertes de Kadiska                     | X          |
+| Target-Statistics | App-Monitoring-Kadiska-Restapi-Target-Statistics-custom | Contrôle les données de performances des targets Kadiska via Rest API | X          |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-kadiska-restapi.md:38]:
```suggestion
| Target-Statistics | App-Monitoring-Kadiska-Restapi-Target-Statistics-custom | Contrôle les données de performance des targets Kadiska via l'API Rest | X          |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-kadiska-restapi.md [hunk 2]
```diff
@@ -3,116 +3,232 @@ id: applications-monitoring-kadiska-restapi
...
-<TabItem value="Service" label="Service">
+<TabItem value="App-Monitoring-Kadiska-Watcher-Restapi-custom" label="App-Monitoring-Kadiska-Watcher-Restapi-custom">
 
-| Nom de la règle                                  | Description                                                                             |
-|:-------------------------------------------------|:----------------------------------------------------------------------------------------|
-| App-Monitoring-Kadiska-Restapi-Target-Statistics | Découverte de targets associées à un runner et supervise ses données de performances.    |
+| Alias                                   | Modèle de service                                                             | Description                                                            |
+|:----------------------------------------|:------------------------------------------------------------------------------|:-----------------------------------------------------------------------|
+| Watcher-Statistics-Per-Site-And-Gateway | App-Monitoring-Kadiska-Restapi-Watcher-Statistics-Per-Site-And-Gateway-custom | Contrôle les données de performances des watchers Kadiska via Rest API |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-kadiska-restapi.md:49]:
```suggestion
| Watcher-Statistics-Per-Site-And-Gateway | App-Monitoring-Kadiska-Restapi-Watcher-Statistics-Per-Site-And-Gateway-custom | Contrôle les données de performance des watchers Kadiska via l'API Rest |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-kadiska-restapi.md [hunk 3]
```diff
@@ -3,116 +3,232 @@ id: applications-monitoring-kadiska-restapi
...
-Rendez-vous sur la [documentation dédiée](/docs/monitoring/discovery/services-discovery)
-pour en savoir plus sur la découverte automatique de services et sa [planification](/docs/monitoring/discovery/services-discovery/#règles-de-découverte).
+> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **App-Monitoring-Kadiska-Watcher-Restapi-custom** est utilisé.
+
+</TabItem>
+<TabItem value="Non rattachés à un modèle d'hôte" label="Non rattachés à un modèle d'hôte">
+
+| Alias                          | Modèle de service                                                    | Description                                                            |
+|:-------------------------------|:---------------------------------------------------------------------|:-----------------------------------------------------------------------|
+| Watcher-Statistics-Per-Country | App-Monitoring-Kadiska-Restapi-Watcher-Statistics-Per-Country-custom | Contrôle les données de performances des watchers Kadiska via Rest API |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-kadiska-restapi.md:58]:
```suggestion
| Watcher-Statistics-Per-Country | App-Monitoring-Kadiska-Restapi-Watcher-Statistics-Per-Country-custom | Contrôle les données de performance des watchers Kadiska via l'API Rest |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-kadiska-restapi.md [hunk 4]
```diff
@@ -3,116 +3,232 @@ id: applications-monitoring-kadiska-restapi
...
+| Kadiska Runners  | Découvre les Runners Kadiska  |
+| Kadiska Watchers | Découvre les Watchers Kadiska |
+
+Rendez-vous sur la [documentation dédiée](/docs/monitoring/discovery/hosts-discovery) pour en savoir plus sur la découverte automatique d'hôtes.
+
+#### Découverte de service
+
+| Nom de la règle                                  | Description                                          |
+|:-------------------------------------------------|:-----------------------------------------------------|
+| App-Monitoring-Kadiska-Restapi-Alert-Rules       | Découvre les règles d'alertes de Kadiska             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-kadiska-restapi.md:81]:
```suggestion
| App-Monitoring-Kadiska-Restapi-Alert-Rules       | Découvre les règles d'alerte de Kadiska             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-kadiska-restapi.md [hunk 5]
```diff
@@ -193,99 +315,534 @@ yum install centreon-plugin-Applications-Monitoring-Kadiska-Restapi
...
+3. Appliquez le modèle d'hôte **App-Monitoring-Kadiska-Restapi-Alerts-custom**. Une liste de macros apparaît. Les macros vous permettent de définir comment le connecteur se connectera à la ressource, ainsi que de personnaliser le comportement du connecteur.
+4. Renseignez les macros désirées. Attention, certaines macros sont obligatoires.
+
+| Macro                  | Description                                                                                           | Valeur par défaut | Obligatoire |
+|:-----------------------|:------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| KADISKAAPIPROTO        | Specify https if needed (Default: 'https')                                                            | https             |             |
+| KADISKAAPIPORT         | Port used (Default: 443)                                                                              | 443               |             |
+| KADISKAAPICLIENTID     | Set client id                                                                                         |                   | X           |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-kadiska-restapi.md:344]:
```suggestion
| KADISKAAPICLIENTID     | Set client ID                                                                                         |                   | X           |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-kadiska-restapi.md [hunk 6]
```diff
@@ -193,99 +315,534 @@ yum install centreon-plugin-Applications-Monitoring-Kadiska-Restapi
...
+3. Appliquez le modèle d'hôte **App-Monitoring-Kadiska-Runner-Restapi-custom**. Une liste de macros apparaît. Les macros vous permettent de définir comment le connecteur se connectera à la ressource, ainsi que de personnaliser le comportement du connecteur.
+4. Renseignez les macros désirées. Attention, certaines macros sont obligatoires.
+
+| Macro                  | Description                                                                                           | Valeur par défaut | Obligatoire |
+|:-----------------------|:------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| KADISKAAPIPROTO        | Specify https if needed (Default: 'https')                                                            | https             |             |
+| KADISKAAPIPORT         | Port used (Default: 443)                                                                              | 443               |             |
+| KADISKAAPICLIENTID     | Set client id                                                                                         |                   | X           |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-kadiska-restapi.md:363]:
```suggestion
| KADISKAAPICLIENTID     | Set client ID                                                                                         |                   | X           |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-kadiska-restapi.md [hunk 7]
```diff
@@ -193,99 +315,534 @@ yum install centreon-plugin-Applications-Monitoring-Kadiska-Restapi
...
+|:-----------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| KADISKAAPIPROTO        | Specify https if needed (Default: 'https')                                                                                                                         | https             |             |
+| KADISKAAPIPORT         | Port used (Default: 443)                                                                                                                                           | 443               |             |
+| GATEWAYNAME            | Display statistics for watchers attached to a particular gateway                                                                                                   |                   |             |
+| KADISKAAPICLIENTID     | Set client id                                                                                                                                                      |                   | X           |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-kadiska-restapi.md:384]:
```suggestion
| KADISKAAPICLIENTID     | Set client ID                                                                                                                                                     |                   | X           |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-kadiska-restapi.md [hunk 8]
```diff
@@ -193,99 +315,534 @@ yum install centreon-plugin-Applications-Monitoring-Kadiska-Restapi
...
+
+</TabItem>
+<TabItem value="Target-Statistics" label="Target-Statistics">
+
+| Macro                   | Description                                                                                                                                                                                               | Valeur par défaut | Obligatoire |
+|:------------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| PERIOD                  |                                                                                                                                                                                                           | 15                |             |
+| TARGETNAME              | Filter to display statistics for particular net tracer targets. Can be a regex or a single tracer target. A target name must be given.  Regex example: --filter-target-name="(mylab.com\|shop.mylab.com)" |                   |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-kadiska-restapi.md:416]:
```suggestion
| TARGETNAME              | Filter to display statistics for particular net tracer targets. Can be a regex or a single tracer target. A target name must be given. Regex example: --filter-target-name="(mylab.com\|shop.mylab.com)" |                   |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-kadiska-restapi.md [hunk 9]
```diff
@@ -193,99 +315,534 @@ yum install centreon-plugin-Applications-Monitoring-Kadiska-Restapi
...
+| WARNINGPACKETSLOSSPRCT  |                                                                                                                                                                                                           |                   |             |
+| CRITICALPACKETSLOSSPRCT | Critical threshold for packets' loss in percentage                                                                                                                                                        |                   |             |
+| WARNINGPATHLENGTH       | Warning threshold for path length to reach targets                                                                                                                                                        |                   |             |
+| CRITICALPATHLENGTH      | Critical threshold for path length to reach targets.  item --warning-packets-loss-prct  Warning threshold for packets' loss in percentage                                                                 |                   |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-kadiska-restapi.md:420]: item --warning-packets-loss-prct?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-kadiska-restapi.md [hunk 10]
```diff
@@ -193,99 +315,534 @@ yum install centreon-plugin-Applications-Monitoring-Kadiska-Restapi
...
+| CRITICALISPUSERS                |                                                                                                   |                   |             |
+| WARNINGISPWAITINGTIMEAVG        |                                                                                                   |                   |             |
+| CRITICALISPWAITINGTIMEAVG       |                                                                                                   |                   |             |
+
+</TabItem>
+<TabItem value="Watcher-Statistics-Per-Site-And-Gateway" label="Watcher-Statistics-Per-Site-And-Gateway">
+
+| Macro                               | Description | Valeur par défaut | Obligatoire |
+|:------------------------------------|:------------|:------------------|:-----------:|
+| PERIOD                              |             | 15                |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-kadiska-restapi.md:494]: no description

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-kadiska-restapi.md [hunk 11]
```diff
@@ -193,99 +315,534 @@ yum install centreon-plugin-Applications-Monitoring-Kadiska-Restapi
...
-    --use-new-perfdata
+	--plugin=apps::monitoring::kadiska::plugin \
+	--mode=watcher-statistics \
+	--client-id='' \
+	--client-secret='' \
+	--select-watcher-name='' \
+	--select-site-name='' \
+	--select-gateway-name='' \
+	--period='15' \
+	--port='' \
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-kadiska-restapi.md:591]: port and proto are empty

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-kadiska-restapi.md [hunk 12]
```diff
@@ -193,99 +322,533 @@ yum install centreon-plugin-Applications-Monitoring-Kadiska-Restapi
...
+| WARNINGPACKETSLOSSPRCT  | Warning threshold for packet loss in percentage                                                                                                                                                                             |             |
+| CRITICALPACKETSLOSSPRCT | Critical threshold for packet loss in percentage                                                                                                                                                        |                   |             |
+| WARNINGPATHLENGTH       | Warning threshold for path length to reach targets                                                                                                                                                        |                   |             |
+| CRITICALPATHLENGTH      | Critical threshold for path length to reach targets.  item --warning-packets-loss-prct  Warning threshold for packets' loss in percentage                                                                 |                   |             |
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-kadiska-restapi.md:427]:
```suggestion
| WARNINGPACKETSLOSSPRCT  | Warning threshold for packet loss in percentage                                                                                                                                                                             |             |
| CRITICALPACKETSLOSSPRCT | Critical threshold for packet loss in percentage                                                                                                                                                        |                   |             |
| WARNINGPATHLENGTH       | Warning threshold for path length to reach targets                                                                                                                                                        |                   |             |
| CRITICALPATHLENGTH      | Critical threshold for path length to reach targets.                                                                  |                   |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-kadiska-restapi.md [hunk 13]
```diff
@@ -193,99 +322,533 @@ yum install centreon-plugin-Applications-Monitoring-Kadiska-Restapi
...
+| --filter-target-name         | Filter to display statistics for particular net tracer targets. Can be a regex or a single tracer target. A target name must be given.  Regex example: --filter-target-name="(mylab.com\|shop.mylab.com)"   |
+| --warning-round-trip         | Warning threshold for round trip in milliseconds.                                                                                                                                                           |
+| --critical-round-trip        | Critical threshold for round trip in milliseconds.                                                                                                                                                          |
+| --warning-path-length        | Warning threshold for path length to reach targets.                                                                                                                                                         |
+| --critical-path-length       | Critical threshold for path length to reach targets.  item --warning-packets-loss-prct  Warning threshold for packets' loss in percentage.                                                                  |
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-kadiska-restapi.md:801]:
```suggestion
| --critical-path-length       | Critical threshold for path length to reach targets.                                                                  |
```

**Review by cg-tw — APPROVED** [review]

**Review by omercier — APPROVED** [review]

**Pull request merged**